src/lib/styles/foundation/partyColors.scss

```diff
@@ -122,6 +162,80 @@
   background-color: var(--noc) !important;
 }
 
+/* BACKGROUND-COLOR 2 -------------------------------------------- */
```

Comment:
I have only added the secondary color as a background class and a fill class - as in my current understanding, this will only occasionally be used in a background capacity. Not as a foreground color. Other classes involving this can be added as needed in future